core/src/test/scripts/test_data/study_es_0/meta_study.txt

```diff
@@ -6,4 +6,5 @@ citation: The Hyve
 pmid: 23000897
 groups: PUBLIC;GDAC;SU2C-PI3K
 short_name: BRCA (TCGA)
-add_global_case_list: true
\ No newline at end of file
+add_global_case_list: true
+tags_file: metadata.yml
```

Comment:
Can you rename metadata.yml to `meta_tags.yml`? I considered `tags.yml` but it would be inconsistent with the other files (only data and meta files), so `meta_tags.yml` would fit right in :)